src/ggrc/converters/handlers/default_people.py

```
@@ -0,0 +1,111 @@
+  """Handler for default verifiers and assessors."""
+
+  KEY_MAP = {
+      "default_assessors": "assessors",
+      "default_verifier": "verifiers",
+  }
+
+  PEOPLE_LABELS_MAP = {
+      display_name.lower(): value
+      for value, display_name
+      in AssessmentTemplate.DEFAULT_PEOPLE_LABELS.items()
+  }
+
+  def _parse_email_values(self):
+    """Parse an email list of default assessors.
+
+    This is the "other" option in the default assessor dropdown menu.
+    """
+    self.add_error("Default people does not support email lists.")
+    # lines = [line.strip() for line in self.raw_value.splitlines()]
+    # value = []
+    # for email in lines:
+    #   if not email:
+    #     continue
+    #   if not Person.is_valid_email(email):
+    #     self.add_warning("invalid email")
+    #   else:
+    #     value.append(email)
+
+    # if not value:
+    #   self.add_error("missing value error")
+    # return value
+
+  def _parse_label_values(self):
+    """Parse predefined default assessors.
+
+    These values are the normal selection in the default assessor dropdown.
+    """
+    value = self.PEOPLE_LABELS_MAP.get(self.raw_value.strip().lower())
+    if not value:
+      self.add_error(errors.WRONG_REQUIRED_VALUE,
+                     column_name=self.display_name,
+                     value=self.raw_value.strip().lower())
+    return value
+
+  def parse_item(self):
+    """Parse values for default assessors."""
+
+    current_app.logger.debug("%s parse item: %s", self.key, self.raw_value)
+
+    if "@" in self.raw_value:
+      return self._parse_email_values()
+    else:
+      return self._parse_label_values()
+
+    current_app.logger.debug("%s parsed value: %s", self.key, self.value)
+
+  def _get_inital_people(self):
+    """Get the default_people dict from current object."""
+    try:
+      value = json.loads(self.row_converter.obj.default_people)
+      if not value:
+        value = {}
+    except TypeError:
+      value = {}
```

Shouldn't this raise some warnings instead of silently replacing value?